dolphin/drivers/dellemc_storage/vmax/alert_handler.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+
+from oslo_log import log
+
+from dolphin import exception
+from dolphin.drivers.dellemc_storage.vmax import alert_mapper
+
+LOG = log.getLogger(__name__)
+
+
+class AlertHandler(object):
+    """Alert handling functions for vmax driver"""
+    default_category = 'New'
+    default_event_type = 'unknown'
+    default_probable_cause = 'Not available'
+    default_severity = 'unknown'
+    default_me_category = 'storage-subsystem'
+    default_location = 'unknown'
+
+    # Mandatory attributes expected in alert info to proceed with model filling
+    mandatory_alert_attributes = ('storage_id', 'storage_name', 'vendor',
+                                  'model', 'connUnitName', 'emcAsyncEventCode')
```

Comment:
plz update nameing style.

Reply:
connUnitName, emcAsyncEventCode these are the part of received trap contents, so used as it is. Otherwise we need to keep internal mapping for all such received field which does not look good.